internal/dev_server/ui/package.json

```diff
@@ -7,22 +7,29 @@
     "dev": "vite",
     "build": "tsc -b && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "prettier:write": "prettier . --write",
```

Comment:
You can run `npm run prettier:write` to format the files in the `internal/dev_server/ui` directory: it's not wired up to anything yet 🤷